Update de.yaml

PuruleZaze opened the pull request:
--> few translations by PuruleZaze

PuruleZaze committed:
Update de.yaml

--> few translations by PuruleZaze

Kravos commented on de.yaml:
```diff
@@ -1,44 +1,44 @@
-2FA_ENABLE_SUCCESS_MSG: 2FA has been enabled successfully
-2FA_HOW_TO_DISABLE: How to disable 2FA?
+2FA_DISABLE: Deaktiviere die Zwei-Faktor-Authentifizierung
+2FA_DISABLE_SUCCESS_MSG: Zwei-Faktor-Authentifizierung wurde erfolgreich deaktiviert
```
```suggestion
2FA_DISABLE_SUCCESS_MSG: Zwei-Faktor-Authentifizierung wurde erfolgreich deaktiviert.
```

Kravos commented on de.yaml:
```diff
@@ -1,44 +1,44 @@
+2FA_DISABLE: Deaktiviere die Zwei-Faktor-Authentifizierung
+2FA_DISABLE_SUCCESS_MSG: Zwei-Faktor-Authentifizierung wurde erfolgreich deaktiviert
+2FA_ENABLE: Aktiviere die Zwei-Faktor-Authentifizierung
+2FA_ENABLE_SUCCESS_MSG: Zwei-Faktor-Authentifizierung wurde erfolgreich aktiviert
```
```suggestion
2FA_ENABLE_SUCCESS_MSG: Zwei-Faktor-Authentifizierung wurde erfolgreich aktiviert.
```

Kravos commented on de.yaml:
```diff
@@ -1,44 +1,44 @@
 2FA_HOW_TO_ENABLE_STEP_3: You will have a 6 digits code. Enter it below.
 2FA_HOW_TO_ENABLE_STEP_4: Click on the button!
-2FA_MANAGE: Manage 2FA
+2FA_MANAGE: Verwalte deine Zwei-Faktor-Authentifizierung
```
```suggestion
2FA_MANAGE: Zwei-Faktor-Authentifizierung Verwaltung
```

Kravos commented on de.yaml:
```diff
@@ -1,44 +1,44 @@
-BACK_TO_LOGIN: Back to login
-BUY_NWC: Buy NWC
+BACK_TO_LOGIN: Zurück zur Anmeldung
+BUY_NWC: Kaufe NWC
```
```suggestion
BUY_NWC: NWC kaufen
```

Kravos commented on de.yaml:
```diff
@@ -1,44 +1,44 @@
 COUPON_INFORMATION:
-COUPON_NOT_VALID: The coupon {coupon} is not valid or has already been used
-COUPON_NWC_ADDED: The amount of {coins} NWC has been added to your account.
+COUPON_NOT_VALID: Dein Gutschein {coupon} ist nicht gültig oder wurde bereits benutzt.
```
```suggestion
COUPON_NOT_VALID: Dein Gutschein {coupon} ist ungültig oder wurde bereits verwendet.
```

Kravos commented on de.yaml:
```diff
@@ -68,50 +68,50 @@ LANGUAGE_FR: French
 LANGUAGE_TR: Turkish
-LOGIN: Login
-LOGOUT: Logout
+LOGIN: Anmeldung
```
```suggestion
LOGIN: Anmelden
```

Kravos commented on de.yaml:
```diff
@@ -68,50 +68,50 @@ LANGUAGE_FR: French
-LOGIN: Login
-LOGOUT: Logout
+LOGIN: Anmeldung
+LOGOUT: Abmeldung
```
```suggestion
LOGOUT: Abmelden
```

Kravos commented on de.yaml:
```diff
@@ -68,50 +68,50 @@ LANGUAGE_FR: French
 SAVE_CHANGES: Save changes
 SELECT_NWC_AMOUNT: Select NWC amount
-SELECT_PURCHASE_METHOD: Select purchase method
+SELECT_PURCHASE_METHOD: Wähle eine Bezahlungsmethode aus
```
```suggestion
SELECT_PURCHASE_METHOD: Wähle eine Bezahlungsmethode aus.
```

Pumba98 reviewed (commented):
Not sure if we should change english abbreviation to full words

PuruleZaze committed:
Update de.yaml

--> corrected few grammatical / spelling mistakes
--> added few more translations

Kravos commented on de.yaml:
```diff
@@ -1,18 +1,18 @@
-2FA_HOW_TO_DISABLE: Wie deaktiviere ich die Zwei-Faktor-Authentifizierung?
-2FA_HOW_TO_DISABLE_STEP_1: Get the 6 digits code on the app you used to enable 2FA and enter it below
-2FA_HOW_TO_DISABLE_STEP_2: Click on the button!
+2FA_DISABLE: Deaktiviere die 2FA
```
2FA deaktivieren

Kravos replied:
```suggestion
2FA_DISABLE: 2FA deaktivieren
```

Kravos commented on de.yaml:
```diff
@@ -1,18 +1,18 @@
+2FA_ENABLE_SUCCESS_MSG: Die 2FA wurde erfolgreich aktiviert.
+2FA_HOW_TO_DISABLE: Wie deaktiviere ich die 2FA?
+2FA_HOW_TO_DISABLE_STEP_1: Get the 6 digits code on the app you used to enable 2FA and enter it below. 
+2FA_HOW_TO_DISABLE_STEP_2: Klicke auf die Schaltfläche!
```
Wir bleiben bei Button :)

Kravos replied:
```suggestion
2FA_HOW_TO_DISABLE_STEP_2: Klicke auf den Button!
```

Kravos commented on de.yaml:
```diff
@@ -1,18 +1,18 @@
-2FA_MANAGE: Verwalte deine Zwei-Faktor-Authentifizierung
+2FA_HOW_TO_ENABLE_STEP_2: Verwende die heruntergeladene App um den QR Code zu scannen.
+2FA_HOW_TO_ENABLE_STEP_3: Du erhälst einen 6-stelligen Code. Trage ihn hier unten ein.
+2FA_HOW_TO_ENABLE_STEP_4: Klicke auf die Schaltfläche!
```
Button

Kravos replied:
```suggestion
2FA_HOW_TO_ENABLE_STEP_4: Klicke auf den Button!
```

PuruleZaze committed:
Update de.yaml

applyed change suggestion

Co-authored-by: Kravos <46578788+Kravos@users.noreply.github.com>

PuruleZaze committed:
Update de.yaml

applyed change suggestion

Co-authored-by: Kravos <46578788+Kravos@users.noreply.github.com>

PuruleZaze committed:
Update de.yaml

applyed change suggestion

Co-authored-by: Kravos <46578788+Kravos@users.noreply.github.com>

WilderPrinz commented on de.yaml:
```diff
@@ -68,50 +68,50 @@ LANGUAGE_FR: French
+SHOP_FILTERS_SEARCH: Suchen
 SHOP_FILTERS_SEARCH_PLACEHOLDER: Enter item name
 SHOP_FILTERS_SORT_BY: Sort by
 SHOP_FILTERS_SORT_BY_ALPHA_ASC: Alphabetical (A-Z)
```
Alphabetisch (A-Z)

WilderPrinz commented on de.yaml:
```diff
@@ -68,50 +68,50 @@ LANGUAGE_FR: French
+SHOP_FILTERS_PRICE: Preis
+SHOP_FILTERS_SEARCH: Suchen
 SHOP_FILTERS_SEARCH_PLACEHOLDER: Enter item name
 SHOP_FILTERS_SORT_BY: Sort by
```
Sortiert bei

WilderPrinz commented on de.yaml:
```diff
@@ -1,60 +1,60 @@
+2FA_ENABLE: Aktiviere die 2FA
+2FA_ENABLE_SUCCESS_MSG: Die 2FA wurde erfolgreich aktiviert.
+2FA_HOW_TO_DISABLE: Wie deaktiviere ich die 2FA?
+2FA_HOW_TO_DISABLE_STEP_1: Get the 6 digits code on the app you used to enable 2FA and enter it below. 
```
```suggestion
2FA_HOW_TO_DISABLE_STEP_1: Hol dir den 6-stelligen Code auf die App mit der du 2Fa aktiviert hast und gebe es unten ein.

WilderPrinz commented on de.yaml:
```diff
@@ -1,60 +1,60 @@
+2FA_HOW_TO_ENABLE_STEP_3: Du erhälst einen 6-stelligen Code. Trage ihn hier unten ein.
+2FA_HOW_TO_ENABLE_STEP_4: Klicke auf den Button!
+2FA_MANAGE: 2FA Verwaltung
 ABOUT: About
```
```suggestion
ABOUT: Über

WilderPrinz commented on de.yaml:
```diff
@@ -1,60 +1,60 @@
+ALL_RIGHTS_RESERVED: Alle Rechte vorbehalten
+AMOUNT: Menge
+ARE_YOU_NEW: Bist du neu hier?
 AUTHORITY: Authority
```
```suggestion
AUTHORITY: Autorität

WilderPrinz commented on de.yaml:
```diff
@@ -1,60 +1,60 @@
+COUPON_INVENTORY: Gutschein Inventar
+COUPON_PLACEHOLDER: Gebe den Gutscheincode ein.
+COUPON_REDEEM: Gutschein einlösen
 CUSTOMIZE: Customize
```
```suggestion
CUSTOMIZE: Anpassen

WilderPrinz commented on de.yaml:
```diff
@@ -68,50 +68,50 @@ LANGUAGE_FR: French
+SHOP_FILTERS_SEARCH: Suchen
 SHOP_FILTERS_SEARCH_PLACEHOLDER: Enter item name
 SHOP_FILTERS_SORT_BY: Sort by
 SHOP_FILTERS_SORT_BY_ALPHA_ASC: Alphabetical (A-Z)
```
```suggestion
SHOP_FILTERS_SORT_BY_ALPHA_ASC: Alphabetisch (A-Z)

Kravos replied:
???

WilderPrinz commented on de.yaml:
```diff
@@ -68,50 +68,50 @@ LANGUAGE_FR: French
+SHOP_FILTERS_PRICE: Preis
+SHOP_FILTERS_SEARCH: Suchen
 SHOP_FILTERS_SEARCH_PLACEHOLDER: Enter item name
 SHOP_FILTERS_SORT_BY: Sort by
```
```suggestion
SHOP_FILTERS_SORT_BY: Sortiert nach

Kravos replied:
Whitespace weg

WilderPrinz commented on de.yaml:
```diff
@@ -68,50 +68,50 @@ LANGUAGE_FR: French
+SHOP_FILTERS_CATEGORIES: Kategorien
+SHOP_FILTERS_PRICE: Preis
+SHOP_FILTERS_SEARCH: Suchen
 SHOP_FILTERS_SEARCH_PLACEHOLDER: Enter item name
```
```suggestion
SHOP_FILTERS_SEARCH_PLACEHOLDER: Name des Gegenstandes eingeben

WilderPrinz commented on de.yaml:
```diff
@@ -68,50 +68,50 @@ LANGUAGE_FR: French
-SELECT_PURCHASE_METHOD: Select purchase method
+SELECT_PURCHASE_METHOD: Wähle eine Bezahlungsmethode aus.
 SEND_RESET_LINK: Send reset link
 SECURITY: Security
```
```suggestion
SECURITY: Sicherheit

Kravos replied:
Sicherheit und ohne Whitespace

WilderPrinz commented on de.yaml:
```diff
@@ -68,50 +68,50 @@ LANGUAGE_FR: French
 SELECT_NWC_AMOUNT: Select NWC amount
-SELECT_PURCHASE_METHOD: Select purchase method
+SELECT_PURCHASE_METHOD: Wähle eine Bezahlungsmethode aus.
 SEND_RESET_LINK: Send reset link
```
```suggestion
SEND_RESET_LINK: Zurücksetzungslink anfordern

Kravos replied:
Zurücksetzungslink anfordern

WilderPrinz commented on de.yaml:
```diff
@@ -68,50 +68,50 @@ LANGUAGE_FR: French
+RESET: Zurücksetzen 
+SAVE: Speichern
 SAVE_CHANGES: Save changes
 SELECT_NWC_AMOUNT: Select NWC amount
```
```suggestion
SELECT_NWC_AMOUNT: NWC Menge aussuchen

WilderPrinz commented on de.yaml:
```diff
@@ -68,50 +68,50 @@ LANGUAGE_FR: French
+REGISTER: Registrieren
+RESET: Zurücksetzen 
+SAVE: Speichern
 SAVE_CHANGES: Save changes
```
```suggestion
SAVE_CHANGES: Änderungen speichern

WilderPrinz commented on de.yaml:
```diff
@@ -68,50 +68,50 @@ LANGUAGE_FR: French
+PROFILE: Profil
+PROFILE_SECTION_TITLE: Mein Profil
 REDEEM: Redeem
 REFRESH_DATA: Refresh data
```
```suggestion
REFRESH_DATA: Daten erneuern.

WilderPrinz commented on de.yaml:
```diff
@@ -68,50 +68,50 @@ LANGUAGE_FR: French
-PROFILE_SECTION_TITLE: My profile
+PROFILE: Profil
+PROFILE_SECTION_TITLE: Mein Profil
 REDEEM: Redeem
```
```suggestion
REDEEM: Einlösen

WilderPrinz commented on de.yaml:
```diff
@@ -68,50 +68,50 @@ LANGUAGE_FR: French
+PURCHASE_LEAVE_MESSAGE: Verlasse diese Seite nicht!
+PURCHASE_SUCCESS: Danke für deinen Einkauf! Deinem Account wurden {amount} NWC hinzugefügt.
 PURCHASE_TOS_MESSAGE: By purchasing, you have read and agreed to our {toslink}.
 PURCHASE_VERIFYING_TRANSACTION_MESSAGE: Verifying the current transaction...
```
```suggestion
PURCHASE_VERIFYING_TRANSACTION_MESSAGE: Überprüfen der aktuellen Transaktion

WilderPrinz commented on de.yaml:
```diff
@@ -68,50 +68,50 @@ LANGUAGE_FR: French
-PURCHASE_SUCCESS: Thanks for your purchase! We added {amount} NWC to your account.
+PURCHASE_LEAVE_MESSAGE: Verlasse diese Seite nicht!
+PURCHASE_SUCCESS: Danke für deinen Einkauf! Deinem Account wurden {amount} NWC hinzugefügt.
 PURCHASE_TOS_MESSAGE: By purchasing, you have read and agreed to our {toslink}.
```
```suggestion
PURCHASE_TOS_MESSAGE: Beim Einkaufen stimmst du unseren {toslink} zu.

WilderPrinz commented on de.yaml:
```diff
@@ -68,50 +68,50 @@ LANGUAGE_FR: French
-PURCHASE_ERRORS_UNDEFINED: An error occurred.
+PURCHASE: Kaufen
+PURCHASE_ERRORS_UNDEFINED: Ein Fehler ist aufgetreten.
 PURCHASE_ERRORS_AMOUNT: Please select an amount.
```
```suggestion
PURCHASE_ERRORS_AMOUNT: Bitte wähle eine Menge aus.

WilderPrinz commented on de.yaml:
```diff
@@ -68,50 +68,50 @@ LANGUAGE_FR: French
+  NEW: Neues Passwort
+  TITLE: Passwort
   UPDATE: Update password
 PRIVACY_POLICY: Privacy Policy
```
```suggestion
PRIVACY_POLICY: Datenschutzbestimmungen
```

WilderPrinz commented on de.yaml:
```diff
@@ -68,50 +68,50 @@ LANGUAGE_FR: French
-  TITLE: Password
+  NEW: Neues Passwort
+  TITLE: Passwort
   UPDATE: Update password
```
```suggestion
    UPDATE: Passwort auf den neusten Stand bringen.

WilderPrinz commented on de.yaml:
```diff
@@ -68,50 +68,50 @@ LANGUAGE_FR: French
   CURRENT: Current password
-  FORGOT: Forgot password?
+  FORGOT: Passwort vergessen?
   FORGOT_INSTRUCTIONS: Enter your email and we will send you instructions to reset your password
```
```suggestion
  FORGOT_INSTRUCTIONS: Gib deine E-Mail ein und wir werden dir Anleitungen schicken um dein Passwort zurückzusetzen.
```

WilderPrinz commented on de.yaml:
```diff
@@ -68,50 +68,50 @@ LANGUAGE_FR: French
-  CONFIRM_NEW: Confirm new password
+  CONFIRM: Bestätige dein Passwort
+  CONFIRM_NEW: Bestätige dein neues Passwort
   CURRENT: Current password
```
```suggestion
  CURRENT: Aktuelles Passwort
```

WilderPrinz commented on de.yaml:
```diff
@@ -68,50 +68,50 @@ LANGUAGE_FR: French
 LOGS_SHOP: Shop Logs
 NO_RECORDS_FOUND: No records found
 NUMBER_OF_CHARACTERS: Number of characters
 PAGINATION_TOTAL_COUNT: Showing from {min} to {max} of {total} entries
```
```suggestion
PAGINATION_TOTAL_COUNT: Anzeigen der {min} zur {max} von {total} Zutritten.
```

WilderPrinz commented on de.yaml:
```diff
@@ -68,50 +68,50 @@ LANGUAGE_FR: French
 LOGS_DONATION: Donation Logs
 LOGS_SHOP: Shop Logs
 NO_RECORDS_FOUND: No records found
 NUMBER_OF_CHARACTERS: Number of characters
```
```suggestion
NUMBER_OF_CHARACTERS: Anzahl der Charaktere 
```

WilderPrinz commented on de.yaml:
```diff
@@ -68,50 +68,50 @@ LANGUAGE_FR: French
 LOGS: Logs
 LOGS_DONATION: Donation Logs
 LOGS_SHOP: Shop Logs
 NO_RECORDS_FOUND: No records found
```
```suggestion
NO_RECORDS_FOUND: Keine Einträge gefunden.
```

Kravos replied:
EINTRÄGE

WilderPrinz commented on de.yaml:
```diff
@@ -68,50 +68,50 @@ LANGUAGE_FR: French
+LOGOUT: Abmelden
 LOGS: Logs
 LOGS_DONATION: Donation Logs
 LOGS_SHOP: Shop Logs
```
```suggestion
LOGS_SHOP: Shop logs
```

Kravos replied:
Nicht übersetzen

WilderPrinz commented on de.yaml:
```diff
@@ -68,50 +68,50 @@ LANGUAGE_FR: French
+LOGIN: Anmelden
+LOGOUT: Abmelden
 LOGS: Logs
 LOGS_DONATION: Donation Logs
```
```suggestion
LOGS_DONATION: Spendenverlauf
```

Kravos replied:
Spendenverlauf

WilderPrinz commented on de.yaml:
```diff
@@ -68,50 +68,50 @@ LANGUAGE_FR: French
-LOGOUT: Logout
+LOGIN: Anmelden
+LOGOUT: Abmelden
 LOGS: Logs
```
```suggestion
LOGS: Protokolle 
```

WilderPrinz commented on de.yaml:
```diff
@@ -68,50 +68,50 @@ LANGUAGE_FR: French
 LANGUAGE_IT: Italian
 LANGUAGE_PL: Polish
 LANGUAGE_TR: Turkish
```
```suggestion
LANGUAGE_TR: Türkisch
```

WilderPrinz commented on de.yaml:
```diff
@@ -68,50 +68,50 @@ LANGUAGE_FR: French
 LANGUAGE_IT: Italian
 LANGUAGE_PL: Polish
```
```suggestion
LANGUAGE_PL: Polnisch
```

WilderPrinz commented on de.yaml:
```diff
@@ -68,50 +68,50 @@ LANGUAGE_FR: French
 LANGUAGE_IT: Italian
```
```suggestion
LANGUAGE_IT: Italienisch
```

WilderPrinz commented on de.yaml:
```diff
@@ -1,60 +1,60 @@
+  TITLE: E-Mailadresse
   UPDATE: Update Email
 ENABLED: Enabled
 GAME_ACCOUNTS: Game Accounts
```
```suggestion
GAME_ACCOUNTS: Spielkonten

Kravos replied:
Spiel Accounts

WilderPrinz commented on de.yaml:
```diff
@@ -1,60 +1,60 @@
+  NEW: Neue E-Mailadresse
+  TITLE: E-Mailadresse
   UPDATE: Update Email
 ENABLED: Enabled
```
```suggestion
ENABLED: Aktiviert
```

WilderPrinz commented on de.yaml:
```diff
@@ -1,60 +1,60 @@
+COUPON_PLACEHOLDER: Gebe den Gutscheincode ein.
+COUPON_REDEEM: Gutschein einlösen
 CUSTOMIZE: Customize
 DISABLED: Disabled
```
```suggestion
DISABLED: Deaktiviert
```

WilderPrinz commented on de.yaml:
```diff
@@ -1,60 +1,60 @@
+COUPON_REDEEM: Gutschein einlösen
 CUSTOMIZE: Customize
 DISABLED: Disabled
 DISCONNECT: Disconnect
```
```suggestion
DISCONNECT: Verbindung abgebrochen
```

WilderPrinz commented on de.yaml:
```diff
@@ -1,60 +1,60 @@
+  CURRENT: Derzeitige E-Mailadresse
+  NEW: Neue E-Mailadresse
+  TITLE: E-Mailadresse
   UPDATE: Update Email
```
```suggestion
  UPDATE: E-Mail aktualisieren.
```

PuruleZaze committed:
Update de.yaml

Co-authored-by: WilderPrinz <82899495+WilderPrinz@users.noreply.github.com>

PuruleZaze committed:
Update de.yaml

Co-authored-by: WilderPrinz <82899495+WilderPrinz@users.noreply.github.com>

PuruleZaze committed:
Update de.yaml

Co-authored-by: WilderPrinz <82899495+WilderPrinz@users.noreply.github.com>

PuruleZaze committed:
Update de.yaml

Co-authored-by: WilderPrinz <82899495+WilderPrinz@users.noreply.github.com>

PuruleZaze committed:
Update de.yaml

Co-authored-by: WilderPrinz <82899495+WilderPrinz@users.noreply.github.com>

PuruleZaze committed:
Update de.yaml

Co-authored-by: WilderPrinz <82899495+WilderPrinz@users.noreply.github.com>

PuruleZaze committed:
Update de.yaml

Co-authored-by: WilderPrinz <82899495+WilderPrinz@users.noreply.github.com>

PuruleZaze committed:
Update de.yaml

Co-authored-by: WilderPrinz <82899495+WilderPrinz@users.noreply.github.com>

PuruleZaze committed:
Update de.yaml

Co-authored-by: WilderPrinz <82899495+WilderPrinz@users.noreply.github.com>
